### PR TITLE
Include lambda zip artifact in repository & enable multiple log groups

### PR DESCRIPTION
**Pull request by HajoAhoMantila**

### CODE REVIEW

**Comment by HajoAhoMantila** [comment]
[Send logs to Papertrail](https://trello.com/c/Vy3bJA1C/779-send-logs-to-papertrail)

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ To use the module in Terraform add, the segment below to your Terraform configur
 ```
 module "cloudwatch-log-group-x-papertrail" {
     source = "github.com/vaana-tech/cloudwatch-logs-to-papertrail//tf_module?ref=GIT_TAG_TO_USE"
-    monitor_log_group_name = "/aws/lambda/my-lambda-function"
+    monitor_log_group_name = ["/aws/lambda/my-lambda-function", "/aws/some/other/log/group/name"]
```
Comment (uhef) [inline README.md]: Should be `monitor_log_group_names` to match the terraform declaration.

File: tf_module/main.tf [hunk 2]
```diff
@@ -1,37 +1,38 @@
...
-        variables = {
-            PAPERTRAIL_HOST = "${var.papertrail_host}"
-            PAPERTRAIL_PORT = "${var.papertrail_port}"
-            PARSE_LOG_LEVELS = "${var.parse_log_levels}"
-        }
+  filename      = "${local.lambda_zip_file}"
+  function_name = "${var.lambda_name_prefix}-papertrail-lambda"
+  handler       = "cloudwatch-papertrail.handler"
+  role          = "${var.lambda_log_role_arn}"
+  description   = "Receives events from a CloudWatch log group and sends them to Papertrail"
```
Comment (uhef) [inline tf_module/main.tf]: CloudWatch log group -> CloudWatch log groups

**Review by uhef — CHANGES_REQUESTED** [review]
Looks nice!

In the future we might consider hosting in https://registry.terraform.io/ as discussed!

File: README.md (hunk 1 above)
Comment (HajoAhoMantila) [reply README.md]: Done

File: tf_module/main.tf (hunk 2 above)
Comment (HajoAhoMantila) [reply tf_module/main.tf]: Done

**Pull request merged**